nautobot/docs/configuration/authentication/sso.md

```diff
@@ -29,12 +29,12 @@ For OpenID connect, you'll need to install the `sso` Python extra.
 $ pip3 install "nautobot[sso]"
 ```
 
-#### SAML
+#### OpenID Connect Dependencies
```

Comment:
Sorry, I think I was confusing in my previous comment. This heading here should be `#### SAML Dependencies` - the heading **before** it, at line 24, should be `#### OpenID Connect Dependencies`. The heading at line 233 below can be changed back to simply `#### SAML`.

Reply:
All good!  I got it updated, thanks for clarifying!